chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,782 @@
-the treatment will not be correlated with anything but the results of that treatment.\cite{duflo2007using}
-Causal inference from randomization therefore depends on a specific counterfactual:
-that the units who received the treatment program
-could just as well have been randomized into the control group.
-Therefore, controlling the exact probability that each individual receives treatment
-is the most important part of a randomization process,
-and must be carefully worked out in more complex designs.
-
-Just like sampling, the simplest form of randomization is a uniform-probability process.\sidenote{
-  \url{https://dimewiki.worldbank.org/Randomization_in_Stata}}
-Sampling typically has only two possible outcomes: observed and unobserved.
-Randomization, by contrast, often involves multiple possible results
-which each represent different varieties of treatments to be delivered;
-in some cases, multiple treatment assignments are intended to overlap in the same sample.
-Complexity can therefore grow very quickly in randomization
-and it is doubly important to fully understand the conceptual process
-that is described in the experimental design,
-and fill in any gaps before implementing it in code.
-The do-file below provides an example of how to implement a simple random assignment of multiple treatment arms. 
-
-\codeexample{simple-multi-arm-randomization.do}{./code/simple-multi-arm-randomization.do}
-
-Some types of experimental designs necessitate that randomization results be revealed in the field.
+Some types of experimental designs necessitate
+that randomized assignment results be revealed in the field.
 It is possible to do this using survey software or live events, such as a live lottery.
 These methods typically do not leave a record of the randomization,
-so particularly when the experiment is done as part of data collection,
-it is best to execute the randomization in advance and preload the results.
-Even when randomization absolutely cannot be done in advance, it is still useful
-to build a corresponding model of the randomization process in Stata
-so that you can conduct statistical analysis later
-including checking for irregularities in the field assignment.
-Understanding that process will also improve the ability of the team
-to ensure that the field randomization process is appropriately designed and executed.
+as such can never be reproducible. 
+However, you can often run your randomization in advance 
+even when you do not have list of eligible units in advance.
+Let's say you want to, at various health facilities, 
+randomly select a sub-sample of patients as they arrive.
+You can then have a pre-generated list 
+with a random order of ''in sample'' and ''not in sample''.
+Your field staff would then go through this list in order
+and cross off one randomized result as it is used for a patient.
+
+This is especially beneficial if you are implementing a more complex randomization,
```

Comment:
the below examples are super but i wonder if should move to wiki? feels like a finer level of detail than other sections.

Reply:
https://github.com/worldbank/dime-data-handbook/issues/467